apps/studio/components/interfaces/Organization/BillingSettings/Subscription/SubscriptionPlanUpdateDialog.tsx

```diff
@@ -568,6 +572,16 @@ const SubscriptionPlanUpdateDialog = ({
                     )}
                 </div>
               )}
+
+              {projects.length === 0 && (
+                <div className="pb-2">
+                  <Admonition title="Empty organization" type="warning">
+                    This organization has no active projects. Did you select the correct
```

Comment:
Do we wanna show the message in case there's no projects at all or when there's no **active** projects? Right now the list of projects passed into this component also includes inactive projects.

Reply:
Good point, filtering now